modules/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java

```diff
@@ -102,59 +102,55 @@ public class AzureBlobStore implements BlobStore {
     private static final int DEFAULT_UPLOAD_BUFFERS_SIZE = (int) new ByteSizeValue(64, ByteSizeUnit.KB).getBytes();
 
     private final AzureStorageService service;
-
     private final BigArrays bigArrays;
+    private final RepositoriesMetrics repositoriesMetrics;
+    private final RepositoryMetadata repositoryMetadata;
 
     private final String clientName;
     private final String container;
     private final LocationMode locationMode;
     private final ByteSizeValue maxSinglePartUploadSize;
 
     private final StatsCollectors statsCollectors = new StatsCollectors();
-    private final AzureClientProvider.SuccessfulRequestHandler statsConsumer;
+    private final AzureClientProvider.RequestMetricsHandler statsConsumer;
```

Comment:
Can we rename this field to `reqestMetricsHandler`? The existing name confuses me.

Reply:
Addressed in b847370b0248d1b6625a7af477b1d0cec896be2e